docs/source/contents/4_draft_mapping.rst

```diff
@@ -917,25 +932,25 @@ Do Not Keep Any Other Values Associated with a Name
 Use Case
 ^^^^^^^^
 
-There are other xpaths in our system that are associated with names that are no longer needed.  Do not migrate these.
+There are other XPaths in our system that are associated with names that are no longer needed.  Do not migrate these.
 
 Justification
 ^^^^^^^^^^^^^
 
-In an RDF based system that leverages linked data, it's unnecessary to keep traditional :code:`mods:name` information
-like authority, displayForm, type, or description. Authorities are present in the URI itself and information such as
-description or displayForm are avaliable from the class our object refers to.  While type is not available, it has little
+In an RDF based system that leverages linked data, it's unnecessary to keep traditional :code:`name` information
+like :code:`authority`, :code:`displayForm`, :code:`type`, or :code:`description`. Authorities are present in the URI itself and information such as
+:code:`description` or :code:`displayForm` are available from the class our object refers to.  While :code:`type` is not available, it has little
 meaning in our current system and will only complicate things in the future.
 
-Xpaths
+XPaths
 ^^^^^^
 
-* :code:`name/role/roleTerm/@authority`
-* :code:`name/@authority`
-* :code:`name/role/roleTerm/@authorityURI`
-* :code:`name/@type`
-* :code:`name/displayForm`
-* :code:`name/description`
+:code:`name/role/roleTerm/@authority` OR
+:code:`name/@authority` OR
+:code:`name/role/roleTerm/@authorityURI` OR
+:code:`name/@type` OR
```

Comment:
I might be missing something, but my understanding is that attributes on a particular element need to be in brackets. In other sections they are represented this way - see line 979. So here, we'd want :code:`name[@type]`, etc.). I'll go through and note each instance where this same thing occurs if we'd like to change it.

Reply:
Thanks for pointing this out; similarly to the comment on approximately line 858, these XPaths point to the attribute(s) themselves. E.g. `name[@type]` we would be saying, "Do not migrate `name` elements with a `@type` attribute!", where `name/@type` says, "Do not migrate the `@type` attribute (and its value)."

Reply:
Thanks for the clarification - no changes are needed. I was definitely "missing something."